hutchresearch/wwu_tinker
# Pip setup complete

You can now communicate with the server via a pip installed tinker.

Test plan:
```
> virtualenv clean_env
> source clean_env/bin/activate
> pip install git+https://www.github.com/hutchresearch/tinker@pip_setup
> python
>>> import tinker
>>>
```
1. Observe tinker imports with no issues.
2. Run a test script such as `test_api.py` while in this virtual environment and observe the server running on the VM gets successfully hit with a request.
